pennylane/optimize/riemannian_gradient.py

```diff
@@ -69,7 +69,6 @@ def append_time_evolution(tape, riemannian_gradient, t, n, exact=False):
         qml.apply(obj)
 
 
-@qml.batch_transform
 def algebra_commutator(tape, observables, lie_algebra_basis_names, nqubits):
```

Comment:
This is just a helper function for the `RiemannianGradientOptimizer` and doesn't need to be a batch transform.

Reply:
Great one less :+1: